### DISCUSSION TITLE
Whole front-end project scrolling bar control

### DISCUSSION THREAD

**Discussion by johnsonwong666** [KingSora/OverlayScrollbars#648] [Q&A]
Thank you open source. I currently want to unify the scrolling bar of the whole front-end project. I want to control all scrollable containers and display the scrollbar when mouse hoover, otherwise I will hide the scroll bar. I wonder if I can use OverlayScrollbars?

**Comment by KingSora** [comment]
Good day @johnsonwong666 :)

Yes, you can use `OverlayScrollbars` for that! You just have to initialize the plugin explicitly to any scrollable container by yourself.
